brian2genn/device.py

```diff
@@ -1257,20 +1259,24 @@ def process_neuron_groups(self, neuron_groups, objects):
             combined_variables = {}
             combined_variable_indices = defaultdict(lambda: '_idx')
             combined_override_conditional_write = set()
-            thresholder_codeobj = getattr(objects.get(obj.name + '_thresholder', None), 'codeobj', None)
-            if thresholder_codeobj is not None:
-                neuron_model.thresh_cond_lines = '_cond'
-            else:
-                neuron_model.thresh_cond_lines = '0'
-
-            for suffix, code_slot in [('_stateupdater', 'stateupdate'),
-                                      ('_thresholder', 'stateupdate'),
-                                      ('_resetter', 'reset'),
-                                      ('_subexpression_update', 'subexpression_update')]:
-                full_name = obj.name + suffix
-                if full_name in objects and objects[full_name].codeobj is not None:
-                    codeobj = objects[full_name].codeobj
-                    combined_abstract_code[code_slot] += [codeobj.abstract_code[None]]
+            has_thresholder = False
+
+            stateupdater_name = obj.name + '_stateupdater'
```

Comment:
The `stateupdater_name` here should always be overwritten in the loop later, right? Maybe make this clearer by setting it to `None` (and possibly use something like `assert stateupdater_name` before creating the `code_object` in the end?).

Reply:
Yeah, that makes sense. I wasn't entirely sure what the significance of the name is downstream from here - it was just the above before, but I figured since we have the _actual_ `StateUpdater`'s name, we might as well use it.

Reply:
Addressed in https://github.com/brian-team/brian2genn/pull/141/commits/145af6f94ce1a558786dae476b0f5342418fd42e